packages/repeater/src/lib/Repeater.ts

```diff
@@ -5,13 +5,22 @@ import {
   RepeaterStatusEvent
 } from '../bus';
 import { RepeaterStatus } from '../models';
-import { Configuration, EventBus } from '@secbox/core';
+import { Configuration, EventBus, Logger } from '@secbox/core';
 import Timer = NodeJS.Timer;
 
+export enum RunningStatus {
+  OFF,
+  STARTING,
+  RUNNING
+}
+
 export class Repeater {
   public readonly repeaterId: string;
+  public runningStatus = RunningStatus.OFF;
```

Comment:
The user should not be able to change `running Status` outside
```suggestion
  private _runningStatus = RunningStatus.OFF;
  
  get runningStatus(): RunningStatus {
    return this._runningStatus;
  }
```